core/shared/src/main/scala/zio/prelude/fx/ZPure.scala

```diff
@@ -495,6 +497,23 @@ sealed trait ZPure[+W, -S1, +S2, -R, +E, +A] { self =>
   final def provideState(s: S1): ZPure[W, Any, S2, R, E, A] =
     set(s) *> self
 
+  /**
+   * Keeps some of the errors, and terminates the fiber with the rest
```

Comment:
Maybe update the documentation here and below? This isn't necessarily even being run on a fiber and we aren't terminating it, just throwing the exception.

Reply:
Silly me... forgot to update comments after copy-paste :smile: